Cargo.toml

```diff
@@ -14,4 +14,4 @@ serde_json = "1.0"
 libc = "0.2"
 jsonpath_lib = { git="https://github.com/gkorland/jsonpath.git", branch="patch-1" }
 redis-module = { version="0.7.0", features = ["experimental-api"]}
-redisearch_api = "0.3.0"
+redisearch_api =  { git="https://github.com/RediSearch/redisearch-api-rs", branch="del_doc"}
```

Comment:
After finishing your `redisearch_api` PR (and releasing) this should be updated to a new release.